app/controllers/sessions.rb

```diff
@@ -9,8 +9,8 @@
     login
```

Comment:
Looking at the entire login route, user email and password were never authenticated to check if the correct user email matched with the correct password. Currently it is only checking to see if that user exists in the database and then if it does then you are logging that user in, without checking for the correct password.